package/batocera/gpu/mali-g31-gbm/mali-g31-gbm.mk

```diff
@@ -24,31 +29,26 @@ endef
 MALI_G31_GBM_POST_INSTALL_STAGING_HOOKS += MALI_G31_GBM_COPY_KHRPLATFORM_STAGING
 endif
 
-ifeq ($(BR2_PACKAGE_BATOCERA_TARGET_ODROIDGOA),y)
-# See https://wiki.odroid.com/odroid_go_advance/application_note/vulkan_on_rk3326
 define MALI_G31_GBM_RK3326_VULKAN_DRIVER_TARGET_32
-	cd $(@D) && \
-	wget https://dn.odroid.com/RK3326/ODROID-GO-Advance/rk3326_r13p0_gbm_with_vulkan_and_cl.zip && \
-	unzip rk3326_r13p0_gbm_with_vulkan_and_cl.zip && \
-	cp $(@D)/libmali.so_rk3326_gbm_arm32_r13p0_with_vulkan_and_cl $(TARGET_DIR)/usr/lib/libmali-bifrost-g31-rxp0-gbm.so && \
-	ln -s /usr/lib/libmali-bifrost-g31-rxp0-gbm.so  $(TARGET_DIR)/usr/lib/libvulkan.so && \
-	ln -s /usr/lib/libmali-bifrost-g31-rxp0-gbm.so  $(TARGET_DIR)/usr/lib/libvulkan.so.1
+       cd $(@D) && \
+       unzip $(MALI_G31_GBM_DL_DIR)/rk3326_r13p0_gbm_with_vulkan_and_cl.zip && \
+       cp $(@D)/libmali.so_rk3326_gbm_arm32_r13p0_with_vulkan_and_cl $(TARGET_DIR)/usr/lib/libmali-bifrost-g31-rxp0-gbm.so && \
+       ln -s /usr/lib/libmali-bifrost-g31-rxp0-gbm.so  $(TARGET_DIR)/usr/lib/libvulkan.so && \
```

Comment:
should also symlink the `libmali*` so and remove the other `so` that is installed by default